addons/l10n_dk/migrations/1.3/end-migrate.py

```diff
@@ -0,0 +1,8 @@
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    for company in env['res.company'].search([('chart_template', '=', 'dk')], order="parent_path"):
```

Comment:
Not that it really plays a role, but why are you ordering specifically by this here?

Reply:
It was because everyone does that but i don't really know if that's helpful tbh ahah, i can remove it if you want

Reply:
No, you're right, cfr https://github.com/odoo/enterprise/commit/42a4cd2e43b993408731db0379fb9d9ea829085e

Reply:
Not that it's important for this script, it would be important if we were to do a try_loading. But it does not hurt much